docs/en/developer.md

```diff
@@ -117,6 +119,32 @@ if (getenv('LDAP_HOSTNAME') && getenv('LDAP_USERNAME') && getenv('LDAP_PASSWORD'
 }
 ```
 
+### Set LDAP as default authenticator
+
+If you’d like to set LDAP as the default authenticator, you will need to reset the list of authenticators first. You can achieve this with the following approach in the `mysite/_config/ldap.yml` file:
+
+```
+---
+Name: authreset
```

Comment:
This block isn't actually required as the `default` authenticator is being specifically overwritten (eg. https://github.com/silverstripe/silverstripe-mfa/blob/master/_config/config.yml#L1-L10)

Reply:
Normally I would agree with you, but when I tried it without the reset block, the member authenticator was still the default, even after /dev/build?flush=1

Reply:
Following kinglozzer's comments, I'll remove the reset block :)